Presentation.Web/app/components/org/structure/org-structure-modal-edit.view.html

```diff
@@ -6,14 +6,11 @@ <h3>Rediger {{ orgUnit.oldName }}</h3>
         <h4>Rediger info</h4>
         <form name="editForm" novalidate="novalidate" class="form-horizontal">
             <div class="form-group">
-                <label for="parent" class="col-xs-3 control-label">
+                <label for="parent-select" class="col-xs-3 control-label">
                     Overordnet enhed
                 </label>
                 <div class="col-xs-8">
-                    <select type="text" data-ui-select2="" name="parent" id="parent" data-ng-model="orgUnit.newParent" class="form-control" data-placeholder="Vælg overordnet" data-ng-required="true" data-ng-disabled="!canChangeParent" tabindex="1">
-                        <option></option>
-                        <option data-ng-repeat="unit in orgUnits" value="{{unit.id}}">{{unit.name}}</option>
-                    </select>
+                    <single-select-select2 id="parent-select" ng-model="parentSelect" ng-disabled="!canChangeParent"/>
```

Comment:
If the parent unit is not selected, we must disable the "Gem" button and a red error text must appear:

"Overordnet enhed skal vælges"

![image](https://user-images.githubusercontent.com/13116889/177094073-fdcf37fe-4696-4b02-b8bc-fd5a2b57d937.png)

Reply:
@a-naskret Scratch that.. we should just remove the option to clear the selection using "false 2in the call to the select2loading service

![image](https://user-images.githubusercontent.com/13116889/177096258-931360a1-24c9-4b4a-99e0-5bf4427472fe.png)